src/components/ebay-notice/examples/02-page-notice-cta/template.marko

```diff
@@ -0,0 +1,12 @@
+<ebay-notice status="attention" type="page" a11y-heading-text="Attention" on-button-click("trigger")>
+    <ebay-notice-content>
+        <p>
+            <strong>Error:</strong> Please take another look at the following:
+        </p>
+        <p>
+            <a href="#">Card number</a>,
+            <a href="#">Expiration date</a>
+        </p>
+    </ebay-notice-content>
+    <ebay-button priority="secondary" transparent>Button</ebay-button>
```

Comment:
For my knowledge, would placing `on-button-click` directly on `ebay-button` also work? If so, what's benefit of putting `on-button-click` on the parent component?

I know we had a discussion around this, but I didn't completely understand the problem at the time.

Reply:
@dkallus Good catch. Yes, I thought the consensus was to place on-button-click directly on the button?

Reply:
Just making sure I understand.
By doing it this way we render all body content from inside `ebay-notice`.  Users can technically add anything they want and we'll render it.  But we're recommending a button. 

The other option was to use `ebay-notice-button` and make it a pseudo attribute of `ebay-notice`.  But in that design we couldn't put the event directly on the button.

This makes sense, though it opens the component up for some abuse.

Reply:
Good catch. It was my mistake. I had it initially on the parent but didnt remove it. 
I will remove it.